gatsby-config.js

```diff
@@ -66,7 +66,7 @@ module.exports = {
               wrapperStyle: 'margin-bottom: 1.0725rem',
             },
           },
-          'gatsby-remark-prismjs',
+          // 'gatsby-remark-prismjs',
```

Comment:
It's probably safe to remove this dependency and smartypants from the `the package.json`

Reply:
Will do in the next PR